src/EditorFeatures/Core.Wpf/InheritanceMargin/InheritanceMarginTag.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis.InheritanceMargin;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.Text.Editor;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InheritanceMargin
+{
+    internal class InheritanceMarginTag : IGlyphTag
+    {
+        /// <summary>
+        /// Margin moniker.
+        /// </summary>
+        public readonly ImageMoniker Moniker;
+
+        /// <summary>
+        /// Members needs to be shown on this line. There might be multiple members.
+        /// For example:
+        /// interface IBar { void Foo1(); void Foo2(); }
+        /// class Bar : IBar { void Foo1() { } void Foo2() { } }
+        /// </summary>
+        public readonly ImmutableArray<InheritanceMemberItem> MembersOnLine;
```

Comment:
do thse need to be public?

Reply:
I am expecting this is consumed by the ViewModel later (Yeah, it doesn't exist) so just mark it as public

Reply:
sgtm.